### PR TITLE
Updated Wormholy from v1.6.4 to v1.6.5

### PR DESCRIPTION
**Pull request by pmusolino**
This PR updates [Wormholy](https://github.com/pmusolino/Wormholy) from version 1.6.4 to version 1.6.5.

## Changes in v.1.6.5
- Fix: crash on iOS 15
- Fix authentication challenge handling to fix SSL pinning issue.
- Fix: keyboard content inset bug.
- Added an option for a default filter in the search bar.

## Testing
- Run `bundle exec pod install`.
- Try to open Wormholy from the Settings screen running the app in debug. It should work like before.

Update release notes:
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -45,7 +45,7 @@ PODS:
   - Sourcery (1.0.3)
   - StripeTerminal (1.4.0)
   - SVProgressHUD (2.2.5)
-  - UIDeviceIdentifier (1.7.1)
+  - UIDeviceIdentifier (1.6.0)
```
Comment (allendav) [inline Podfile.lock]: Why did this down rev? See https://github.com/woocommerce/woocommerce-ios/pull/5319/files#diff-e5b768ab2330ee29074f2eeadddd406f19e73b14673eb5f9b5ed6b5b77f4e363R48

**Review by allendav — CHANGES_REQUESTED** [review]
Tests fine. Tested on both an iPad running 14.8 and an iPhone running 15.1. Please double check the downrevving of UIDeviceIdentifier

**Comment by pmusolino** [comment]
Good catch @allendav. Not sure why this happened, but a `pod repo update` before a `pod install` solved the issue.

**Comment by allendav** [comment]
> Good catch @allendav. Not sure why this happened, but a pod repo update before a pod install solved the issue.

I still see 1.6.0 in the Podfile.lock here (and on my machine too even after PRU/PI)?

Edit: On my machine, after running `bundle exec pod update UIDeviceIdentifier` I get the expected diff in Podfile.lock:

```
diff --git a/Podfile.lock b/Podfile.lock
index 073cf59e5..262a44e06 100644
--- a/Podfile.lock
+++ b/Podfile.lock
@@ -45,7 +45,7 @@ PODS:
   - Sourcery (1.0.3)
   - StripeTerminal (2.3.0)
   - SVProgressHUD (2.2.5)
-  - UIDeviceIdentifier (1.6.0)
+  - UIDeviceIdentifier (1.7.1)
   - WordPress-Aztec-iOS (1.11.0)
   - WordPress-Editor-iOS (1.11.0):
     - WordPress-Aztec-iOS (= 1.11.0)
@@ -178,7 +178,7 @@ SPEC CHECKSUMS:
   Sourcery: 70a6048014bd4f37ea80e6bd4354d47bf3b760e1
   StripeTerminal: c53f6bc71c11b11867992478b1806a7c6ec16c0f
   SVProgressHUD: 1428aafac632c1f86f62aa4243ec12008d7a51d6
-  UIDeviceIdentifier: f4bf3b343581a1beacdbf5fb1a8825bd5f05a4a4
+  UIDeviceIdentifier: d0fee204f467dacf8808b7ac14ce43affeb271a5
   WordPress-Aztec-iOS: 050b34d4c3adfb7c60363849049b13d60683b348
   WordPress-Editor-iOS: 304098424f1051cb271546c99f906aac296b1b81
   WordPressAuthenticator: 06278534519c741ecea346f504ad8d08082bfb0d
```

**Comment by pmusolino** [comment]
@allendav after doing `bundle exec pod update UIDeviceIdentifier`, everything seems to be solved. Can you check it please? 🙇

**Comment by allendav** [comment]
Looking now

**Review by allendav — APPROVED** [review]
:shipit:

**Pull request merged**